lib/sensu-plugins-http/common.rb

```diff
@@ -0,0 +1,30 @@
+
+module Common
+  def initialize
+    super()
+    aws_config
+  end
+
+  def aws_config
+    Aws.config.update(
```

Comment:
I was using the same approach on another sensu check so I've just been making that common function generic if I need to pull secure configuration from an AWS bucket.